wsmaster/che-core-api-auth-bitbucket/src/main/java/org/eclipse/che/security/oauth1/NoopOAuthAuthenticator.java

```diff
@@ -49,7 +49,6 @@ public String computeAuthorizationHeader(String userId, String requestMethod, St
 
   @Override
   public String getLocalAuthenticateUrl() {
-    throw new RuntimeException(
-        "The fallback noop authenticator cannot be used for authentication. Make sure OAuth is properly configured.");
+    return "Noop URL";
```

Comment:
should be const. In general, why do we need a dummy implementation in the case if no Bitbucket Server
integration is configured? Could you please add more details in the Javadocs for this class?

Reply:
> should be const.

This string is used only once, not sure we need a separate const for that.
> In general, why do we need a dummy implementation in the case if no Bitbucket Server
integration is configured? Could you please add more details in the Javadocs for this class?

It is needed to initialise an empty `BitbucketServerApiClient` which returns `false` on `isConnected()` reqests. This is needed to check if bitbucket oAuth is configured or not. Updated the javadoc.